ush/forecast_postdet.sh

```diff
@@ -797,57 +786,77 @@ MOM6_out() {
     export COMIN="$ROTDIR/$CDUMP.$PDY/$cyc"
     export COMOUT="$ROTDIR/$CDUMP.$PDY/$cyc"
   fi
-  [[ ! -d $COMOUT ]] && mkdir -m 775 -p $COMOUT
+  [[ ! -d $COMOUT/ocean ]] && mkdir -m 775 -p $COMOUT/ocean
 
-  if [ $inistep = 'cold' ]; then
-    cp $DATA/ufs.cpld.cold.cpl.r.*.nc $COMOUT/
-    cp $DATA/rpointer.cpl $COMOUT/
+  if [ ${inistep:-"restart"} = 'cold' ]; then
+    $NLN $COMOUT/ocean/ufs.cpld.cold.cpl.r.*.nc $DATA/ufs.cpld.cold.cpl.r.*.nc
+    $NLN $COMOUT/ocean/rpointer.cpl $DATA/rpointer.cpl
     status=$?
     exit $status
   else
-    if [ $FHRGRP -eq 0 ]; then
-      fhrlst="anl"
-    else
-      fhrlst=$OUTPUT_HF
-    fi
+    fhrlst=$OUTPUT_FH
 
-    # copy ocn files
     for fhr in $fhrlst; do
-      export fhr=$fhr
-      if [[ 10#$fhr -ge 6 ]]; then
-        hh_inc_m=$((10#$FHOUT/2))
-        hh_inc_o=$((10#$FHOUT  ))
-
-        # ------------------------------------------------------
-        #  adjust the dates on the mom filenames and save
-        # ------------------------------------------------------
-        VDATE=$($NDATE $fhr $IDATE)
-        YYYY=`echo $VDATE | cut -c1-4`
-        MM=`echo $VDATE | cut -c5-6`
-        DD=`echo $VDATE | cut -c7-8`
-        HH=`echo $VDATE | cut -c9-10`
-        SS=$((10#$HH*3600))
-
-        m_date=$($NDATE -$hh_inc_m $VDATE)
-        p_date=$VDATE
-
-        year=`echo $m_date | cut -c1-4`
-        month=`echo $m_date | cut -c5-6`
-        day=`echo $m_date | cut -c7-8`
-        hh=`echo $m_date | cut -c9-10`
-
-        export ocnfile=ocn_${year}_${month}_${day}_${hh}.nc
-
-        echo "$NCP -p $ocnfile $COMOUT/ocn$p_date.$ENSMEM.$IDATE.nc"
-        $NCP -p $ocnfile $COMOUT/ocn$p_date.$ENSMEM.$IDATE.nc
+      if [ $fhr = 'anl' ]; then
```

Comment:
Suggestion, may be for future if it is too much in this PR.
Can we move this to an e.g. `forecast_postdet_ocean.sh`?